src/app/zeko/circuits/rule_commit.ml

```diff
@@ -166,9 +166,7 @@ struct
       in
       let input =
         let open Random_oracle.Input.Chunked in
-        append
```

Comment:
I'm not sure this is sound because the order in which you insert accounts into the ledger isn't deterministic, so you can first insert an account into index 1, then index 0, which will probably result in the acc set being ordered inversely of the ledger.

Reply:
You are signing the ledger, not its history.